fastlane/Fastfile

```diff
@@ -245,6 +245,13 @@ platform :ios do
     }
   end
 
+  desc "Generate Jazzy Docs"
+  lane :generate_docs do
+    Dir.chdir("..") do
+      sh("./scripts/generate_docs.sh")
```

Comment:
we could just run this directly, but I figured having a lane would be a nice convenience